pkg/build/image_builder.go

```diff
@@ -73,6 +73,10 @@ func (bc *Context) BuildImage() error {
 		return errors.Wrap(err, "failed to fixate apk world")
 	}
 
+	if err := bc.normalizeApkScriptsTar(); err != nil {
+		return err
```

Comment:
Should this be a warning ?

Reply:
No, if it fails to normalize the `scripts.tar`, we should consider it an error.